From eaac2d2d22388209ba36427e8bed7b9fa8c28557 Mon Sep 17 00:00:00 2001
From: Jose Quaresma <quaresma.jose@gmail.com>
Date: Thu, 7 Sep 2023 10:50:00 +0100
Subject: [PATCH] msi-ec: Add Summit E14Evo new firmware

Added a new firmware version: 14F1EMS1.116

On latest BIOS version released on 03/07/2023 msi_ec is not allowed to load.

$ modprobe -v msi_ec
modprobe: ERROR: could not insert 'msi_ec': Operation not supported

$ journalctl -k | grep "msi_ec\|E14F1IMS"
kernel: DMI: Micro-Star International Co., Ltd. Summit E14Evo A12M/MS-14F1, BIOS E14F1IMS.114 03/07/2023
kernel: msi_ec: Firmware version is not supported: '14F1EMS1.116'

Signed-off-by: Jose Quaresma <quaresma.jose@gmail.com>
---
 msi-ec.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/msi-ec.c b/msi-ec.c
index 4748036..d8a21b7 100644
--- a/msi-ec.c
+++ b/msi-ec.c
@@ -689,6 +689,7 @@ static struct msi_ec_conf CONF7 __initdata = {
 
 static const char *ALLOWED_FW_8[] __initconst = {
 	"14F1EMS1.115",
+	"14F1EMS1.116",
 	NULL
 };
 
-- 
2.42.0

